MapleServer2/Types/Player.cs

```diff
@@ -167,7 +168,7 @@ public Player(long accountId, long characterId, string name, byte gender, Job jo
             Titles = new List<int>();
             ChatSticker = new List<ChatSticker>();
             FavoriteStickers = new List<int>();
-            Emotes = new List<int>();
+            Emotes = new List<int>() { 90200011, 90200004, 90200024, 90200041, 90200042, 90200057, 90200043, 90200022, 90200031, 90200005, 90200006, 90200003, 90200092, 90200077, 90200073, 90200023, 90200001, 90200019, 90200020, 90200021 };
```

Comment:
All new characters have these emotes?

Reply:
yeah, only about 4 of them are unlocked for new characters. The rest are locked behind trophies